### ISSUE TITLE
nan loss

### ISSUE THREAD

**Issue by Huiimin5** [biomedia-mira/stochastic_segmentation_networks#3]
Hi, thank you so much for your elegant implementation. When trying to reproduce your results in the paper, I met nan loss problems after 2 epoches' training.
![image](https://user-images.githubusercontent.com/11154631/112249380-a1a52280-8c92-11eb-965a-611b51d653d3.png)
I have checked environmental settings and they follow requirements.txt. May I get some help from you?
Btw, I keep getting this warning:
projects/stochastic_segmentation_networks/ssn/nifti/datasets.py:115: UserWarning: The given NumPy array is not writeable, and PyTorch does not support non-writeable tensors. This means you can write to the underlying (supposedly non-writeable) NumPy array using the tensor. You may want to copy the array to protect its data or make it writeable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at  /opt/conda/conda-bld/pytorch_1595629403081/work/torch/csrc/utils/tensor_numpy.cpp:141.)
  target = torch.tensor(target, dtype=target_type) if target is not None else nan_tensor

Is this where the problem comes from?

**Comment by MiguelMonteiro** [comment]
Hello, I am not sure if the two issues are related. Let me have a look at the NaN loss and get back to you. I think the warning comes from the newer version of torch. What version of Pytorch are you using? Are you using the config that was provided or did you make any changes?

**Comment by Huiimin5** [comment]
Sorry about the late reply. The problem is caused by the dataset. In Brats2017 one data is empty and a directory with the same name containing the real image information. I just compressed that data into .gz version in replace of the original empty data. Thank you a lot for your response.

**Comment by MiguelMonteiro** [comment]
Glad to see it's fixed :)
